tests/unit/s2n_client_cert_verify_test.c

```diff
@@ -42,6 +46,86 @@ static int test_sign(const struct s2n_pkey *priv_key, s2n_signature_algorithm si
     return S2N_SUCCESS;
 }
 
+struct task_params {
+    struct s2n_connection *conn;
+    struct s2n_async_pkey_op *op;
+};
```

Comment:
Do you need this structure? The op is now a global

Reply:
Removed this and made conn object global too